assets/js/src/core/modules/execution-engine/jobs/default/notification-job-container.tsx

```diff
@@ -79,12 +79,14 @@ export const NotificationJobContainer = (props: JobProps): React.JSX.Element =>
 
   function messageHandler (event: MessageEvent): void {
     const data: any = JSON.parse(event.data as string)
+    console.log(data)
 
     if (data.jobRunId !== jobId.current) {
       return
     }
 
     if (data.progress !== undefined) {
+      console.log(data.progress)
```

Comment:
should we remove this?

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/564/commits/5072b214376cca142a008905c08b9b35076a0f7b and https://github.com/pimcore/studio-ui-bundle/pull/564/commits/ed3f21d5edd5f039e5e8886539d20fbdf2002c74